src/client/Microsoft.Identity.Client/AuthScheme/TokenType.cs

```diff
@@ -6,7 +6,7 @@ namespace Microsoft.Identity.Client.AuthScheme
     /// <summary>
     /// Specifies the token type to log to telemetry.
     /// </summary>
-    internal enum TokenType
+    public enum TokenType
```

Comment:
As stated before, we can't have an enum for this. It has to be an `int` or smth.

Reply:
or static strings?